### DISCUSSION TITLE
In the case of mouse hover or breakpoint

### DISCUSSION THREAD

**Discussion by 1715526759** [tailwindlabs/tailwindcss#11729] [Ideas]
For example, if I want to write multiple styles in the medium window size, I need to write multiple md prefixes, can I write multiple styles after only one md prefix?

**Comment by wongjn** [comment]
There have been suggestions about adding such a feature. The maintainers have addressed this request before, that you may wish to read up on if you have not already done so:
- [A Twitter thread from the creator of Tailwind, Adam Wathan](https://twitter.com/adamwathan/status/1461519820411789314)
- [A previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611)
